sdks/java/core/src/main/java/org/apache/beam/sdk/util/RowJson.java

```diff
@@ -351,6 +351,9 @@ private void writeValue(JsonGenerator gen, FieldType type, Object value) throws
         case ROW:
           writeRow((Row) value, type.getRowSchema(), gen);
           break;
+        case LOGICAL_TYPE:
+          writeValue(gen, type.getLogicalType().getBaseType(), value);
+          break;
```

Comment:
Nice! For some reason I was thinking this would be a big pain, but this is nice and simple :+1: 

Could you add some tests of logical types to `RowJsonTest`? You could just use `LogicalTypes.FixedSizeBytes` I think.

Probably the easiest way is to add a `makeLogicalTypeTestCase` here: https://github.com/apache/beam/blob/35da90a94953597e9e5676e1e1e70f27d2a8f064/sdks/java/core/src/test/java/org/apache/beam/sdk/util/RowJsonTest.java#L68-L73

Reply:
Added `makeLogicalTypeTestCase`.